improtreskApi/rest_router.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import routers
+
+from api.rest import serializers
+from api_textual.rest import texts
+
+router = routers.DefaultRouter()
+router.register(r'texts', texts.TextViewSet, base_name="text")
```

Comment:
@PetrDlouhy, it is not necessary to provide `base_name` when `queryset` is defined as class property. Is there any advantage in defining it?